script/bootstrap

```diff
@@ -30,6 +30,19 @@
 #       This script resolves all dependencies.
 #
 
+_checkfile=./src/agent/main.cpp
+if [ ! -f $_checkfile ]
```

Comment:
It would seem so - but - the problem is, if the user did not start the script correctly then this script cannot find the _initrc file

for example if the user does this:
```
bash$ cd ${borderrouter}
bash$ cd ${scripts}
bash$ sh ./bootstrap
```
Then the new path to `_initrc` is:  `./_initrc`,  and is no longer `script/_initrc`

Sort of like a chicken & egg problem.

Reply:
`_initrc` is under the same path of the running script. So in the example, the `_initrc` will be included as `./_initrc` because `"$(dirname $"0")"` would be `.`. I also noticed in `_initrc` it will change the directory to the root of this project. See the [code](https://github.com/openthread/borderrouter/blob/master/script/_initrc#L31). Is that code failing in some cases so that we have to restrict starting directory of this script?

Reply:
you are correct - this safety test does not need to exist
i'll remove it.